docs/api-reference/next/image.md

```diff
@@ -43,17 +47,94 @@ function Home() {
 export default Home
 ```
 
-`Image` accepts the following props:
+## Props
+
+The `Image` component accepts the following properties.
+
+### src
+
+The path or URL to the source image. This is required.
+
+### width
+
+The width of the image, in pixels. Must be an integer without a unit.
+
+Required unless [layout="fill"`](#layout).
+
+### height
+
+The height of the image, in pixels. Must be an integer without a unit.
+
+Required unless [layout="fill"`](#layout).
+
+### layout
+
+The layout behavior of the image as the viewport changes size.
+
+When `fixed`, the image dimensions will not change as the viewport changes (no responsiveness) similar to the native `<img>` element.
+
+When `intrinsic`, the image will scale the dimensions down for smaller viewports but maintain the original dimensions for larger viewports.
+
+When `responsive`, the image will scale the dimensions down for smaller viewports and scale up for larger viewports.
+
+When `fill`, the image will stretch both width and height to the dimensions of the parent element, usually paired with [object-fit](https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit).
+
+Default `intrinsic`.
```

Comment:
Maybe replace this with `Defaults to ___` here and in other similar places.

Reply:
Sounds good, I updated all the Default usage on this page